src/components/TitleTransferPanel/TitleView/Holder.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import css from "./TitleView.module.scss";
+import { AddressInfo } from "../AddressInfo";
+import { ExternalLink } from "../../UI/ExternalLink";
+import { makeEtherscanAddressURL } from "../../../utils";
+
+interface HolderProps {
+  holderAddress: string;
+}
+
+export const Holder = ({ holderAddress }: HolderProps) => {
```

Comment:
Hmmm think we spoke about combining `Holder` and `Beneficiary` and differentiate by the prop right?